gradle/verification-metadata.xml

```diff
@@ -1112,6 +1112,14 @@
             <sha256 value="298f0c2e979532171a05a41603263733b65f89113d4e809ccd0e26990149de18" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.gradleup.shadow" name="shadow-gradle-plugin" version="8.3.0">
+         <artifact name="shadow-gradle-plugin-8.3.0.jar">
+            <md5 value="d481a72bf8ce2d0e0a3b15d9190518c1" origin="Generated by Gradle"/>
+            <sha256 value="ffcb98f1d14698ee45e3aad02755281bea0617d3f8d3ae20047f54ad9c4f5a4b" origin="Generated by Gradle">
```

Comment:
This hash points to another artifact causing CI failures and should be updated accordingly to [doc](https://github.com/JetBrains/kotlin/tree/master?tab=readme-ov-file#dependency-verification)

Reply:
Thanks for the heads-up! Done.